### ISSUE TITLE
How should the filelist look like?

### ISSUE THREAD

**Issue by Cristy94** [JiahuiYu/generative_inpainting#20]
First I want to say congratulations for the amazing job you did!

I did not find any example of what the `Prepare training images filelist.` step consists on. How must the images be listed? What is the format? One file name per line, like this?

```
pic-1.jpg
pic-2.jpg
```

**Comment by JiahuiYu** [comment]
Please refer to #15 .

**Comment by Cristy94** [comment]
Thank you, I just saw that closed issue, you might want to add a link from the readme to it.

Why are there two lines in the dataset?

```
  celebahq: [
    'data/celeba_hq/train_shuffled.flist',
    'data/celeba_hq/train_shuffled.flist'
  ]
```

**Comment by JiahuiYu** [comment]
Sorry, typo. The second flist is for validation. I've made an update.

**Comment by Cristy94** [comment]
So I have to split myself the dataset into test/validation? What does static_view mean?

**Comment by JiahuiYu** [comment]
static_view is for tensorboard. If static_view is true, same masks (otherwise random masks) will be used for validation images.

**Comment by Cristy94** [comment]
Cool, thanks a lot. Can I disable static view from the .yml and have random masks instead? I only see a `STATIC_VIEW_SIZE: 30` setting.

**Comment by lijhong** [comment]
Is the training set and validation set the same picture?

**Comment by JiahuiYu** [comment]
They are not.

**Comment by Cristy94** [comment]
The validation set is the one used for the network loss output?
